response.go

```diff
@@ -24,12 +24,10 @@ type Response struct {
 
 //NewResponseFromReader returns a Response or error based on the given reader.
 func NewResponseFromReader(r io.Reader) (*Response, error) {
-	buf := &bytes.Buffer{}
-
```

Comment:
Cleaned up old unneeded byte copy here -- I used to have the original raw bytes saved in order to work on signing, but that's all taken care of now.